integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodsShutdownOption.java

```diff
@@ -99,15 +114,20 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
     // get a unique operator namespace
     logger.info("Getting a unique namespace for operator");
     assertNotNull(namespaces.get(0), "Namespace list is null");
-    String opNamespace = namespaces.get(0);
+    opNamespace = namespaces.get(0);
 
     // get a unique domain namespace
     logger.info("Getting a unique namespace for WebLogic domain");
     assertNotNull(namespaces.get(1), "Namespace list is null");
     domainNamespace = namespaces.get(1);
 
-    // install and verify operator
-    installAndVerifyOperator(opNamespace, domainNamespace);
+    // install and verify operator with FINE logging level
+    HelmParams opHelmParams =
```

Comment:
Why we need a FINE logging level explicitly ?  It will generate a ton of message.